pkg/client/connector/service.go

```diff
@@ -299,6 +299,29 @@ func (s *service) GetCloudAPIKey(ctx context.Context, req *rpc.KeyRequest) (*rpc
 	return &rpc.KeyData{ApiKey: key}, nil
 }
 
+func (s *service) GetCloudLicense(ctx context.Context, req *rpc.LicenseRequest) (*rpc.LicenseData, error) {
+	ctx = s.callCtx(ctx, "GetCloudLicense")
+	license, err := s.getCloudLicense(ctx, req.GetId())
+	if err != nil {
+		return nil, err
+	}
+	return &rpc.LicenseData{License: license}, nil
+}
+func (s *service) getCloudLicense(ctx context.Context, id string) (string, error) {
```

Comment:
Is there a reason to have `.getCloudLicense` separate from `.GetCloudLicense`?  The reason I have a separate `.getCloudAPIKey` is because `traffic_manager.go` wants to use it (and the separate `.getCloudAccessToken` is vestigial; a few different things _used to_ want to use it); I don't see anywhere else that wants to use `.getCloudLicense`.